lib/Rx/Operator/SkipWhileOperator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Rx\Operator;
+
+use Rx\ObservableInterface;
+use Rx\Observer\CallbackObserver;
+use Rx\ObserverInterface;
+use Rx\SchedulerInterface;
+
+class SkipWhileOperator implements OperatorInterface
+{
+    /** @var callable */
+    private $predicate;
+
+    /** @var int */
+    private $index;
+
+    /** @var bool */
+    private $lastResult;
+
+    public function __construct(callable  $predicate)
+    {
+        $this->predicate  = $predicate;
+        $this->index      = 0;
+        $this->lastResult = false;
```

Comment:
Never mind. Just chatted with @davidwdan on IRC. `lift()` only takes a _factory_ so you always get a new operator. Unless you go full clowntown of course, but then you deserve it? :)